modules/ui/rapid_feature_toggle_dialog.js

```diff
@@ -0,0 +1,176 @@
+import {
+    event as d3_event,
+    select as d3_select 
+} from 'd3-selection';
+
+import { t } from '../util/locale';
+import { icon } from './intro/helper';
+import { uiModal } from './modal';
+export function uiRapidFeatureToggleDialog(context, AIFeatureToggleKey, featureToggleKeyDispatcher) {
```

Comment:
style nit: two blank lines above the `export function...` seems to be the convention.